client/components/jetpack/threat-item-header-new/style.scss

```diff
@@ -23,6 +23,7 @@
     font-weight: 600;
     font-size: $font-body-large;
     line-height: 24px;
+    overflow: scroll;
```

Comment:
`overflow: scroll` causes scrollbars to always appear, even if the filename isn't long enough to require them:
<img width="654" alt="image" src="https://user-images.githubusercontent.com/2012803/148064131-708afb98-40ef-4841-be22-d6fc69f3391b.png">

Using `overflow: auto` works a little better, because it only displays the scrollbars that are necessary, but I think it still looks a little strange having just that element scroll:
<img width="761" alt="image" src="https://user-images.githubusercontent.com/2012803/148064577-6b5ec298-5c18-4151-9907-4ee8406ffa16.png">

Did you consider using `text-overflow: ellipsis` to keep the text to a single line and cut it off with a `...` if it gets too long?
```
white-space: nowrap;
overflow: hidden;
text-overflow: ellipsis;
```
<img width="753" alt="image" src="https://user-images.githubusercontent.com/2012803/148064732-6674a528-eb12-4c94-9db5-1b29e7c40403.png">

Reply:
Much much better! Thanks for the suggestion!